sleuth_crawler/scraper/scraper/spiders/parsers/utils.py

```diff
@@ -19,7 +19,7 @@ def strip_content(data):
         return lines
     except Exception:
         # if page is not a webpage, catch errors on attempted parse
-        return None
+        return [""]
```

Comment:
Why not return an empty list?

Reply:
True, empty lines get discarded anyway in `pipeline`